packages/form-js-playground/src/Playground.js

```diff
@@ -36,7 +37,9 @@ export default function Playground(options) {
 
   container.classList.add('fjs-pgl-parent');
 
-  parent.appendChild(container);
+  if (parent) {
```

Comment:
This means that we should update the type definition so that `container` is optional.

Reply:
Here: https://github.com/bpmn-io/form-js/pull/298/commits/5c712c143cf4e9a5b8b3b16a5562a3a46f0e5b96#diff-a7a6e2acf8e80a6b7daf321655518c1fc00e22b1b9570b93ce805db0937db1dcR11

Reply:
(I can't add suggestion via GH)